### ISSUE TITLE
Talkback behaves strangely with horizontal calendar layout

### ISSUE THREAD

**Issue by citrusella** [kizitonwose/CalendarView#367]
I was testing Android's screen reader with the playing-around app I'm using to learn my way around all of the parts of this, and I noticed if I'm using a horizontal calendar layout with Talkback's swipe gestures to navigate and reach the end of a month, it tries to go to the next month without properly scrolling to it and only sees the first column of days in the next month (in my case Sunday) until it reaches the end of the column, at which point it reads the next day on the list (the last day in the second column) and properly scrolls to the new month (or if it's not paged then it scrolls to the next column and so can get increasingly off-center this way).

I don't know if this is something I just haven't learned how to fix yet (i.e. I haven't figured out how to code with that in mind) or a problem with the library itself. It doesn't happen with vertical scrolling.

**Comment by citrusella** [comment]
Hmm... I tested out the sample app and don't have the same problems on most of those layouts so maybe it's just something I haven't accounted for...

**Comment by kizitonwose** [comment]
I think this can be closed. Without access to the project you are working on, it is hard for me to tell what is happening. Since it cannot be reproduced in the sample app, then there is little I can do. Hopefully, you figured it out.

**Comment by citrusella** [comment]
I didn't figure it out so much as abandon horizontal scroll (largely because a bunch of other issues with what I was coding (unrelated to the calendar view) cropped up that I needed to fix so investigating lost priority), but this getting closed was a reminder I should probably put a pin in figuring out what was going on...

**Comment by kizitonwose** [comment]
Alright, keep me posted 👍

**Comment by citrusella** [comment]
I still haven't figured out what I might have been doing wrong to cause the issue (way more pressing issues with a different part still)... but I did figure out just now that example 5 in the sample app has the strange Talkback behavior problem. (I was looking at it so I could see if something in that example helps me work through something else I'm doing and got the idea to try Talkback with it at the same time.)

(By contrast, example 1 does not have this issue.)

**Comment by citrusella** [comment]
So just for a last update from me on this, probably: I updated to 2.0.1 on a whim today (I was avoiding having to work on a different thing in my app's code 😅) and did my best to get most things working with the new changes to the library (which went swimmingly considering I've never really updated a library that I needed to make changes to before!).

I decided to switch back to horizontal scroll because I didn't want to try figuring out today why I was seeing multiple months onscreen now and figured it'd be a chance to see if the TalkBack behavior was still happening... *and* it wasn't! TalkBack gesture swipes to move from the end of a month to the start of the next one *seem* to properly scroll to the next month instead of causing months to get offset in 2.0.1!

~~Now to just figure out why I'm not seeing month titles anymore...~~
